sources/packages/web/src/components/generic/StatusBadge.vue

```diff
@@ -16,6 +16,12 @@ status Badge
 <script lang="ts">
 import { ref, onMounted, watch } from "vue";
 import { GeneralStatusForBadge } from "@/types";
+import {
+  COLOR_BLACK,
```

Comment:
do we have this constant already in our code? because I do not find finding it in this PR. Am i missing anything